package-res/resources/web/prompting/PromptPanel.js

```diff
@@ -339,7 +339,7 @@ define(['cdf/lib/Base', 'cdf/Logger', 'dojo/number', 'dojo/i18n', 'common-ui/uti
         dashboard: undefined,
 
         parametersChanged: false,
-        onParameterChanged: null,
+        onParameterChanged: {},
```

Comment:
I prefer the previous initial value as null because is we do not register callbacks, will be checked only first if statement in 'parameterChanged' method.
Also IMO need add property description (js doc) for 'onParameterChanged' as for other class properties such as 'onBeforeRender'.